### ISSUE TITLE
Can't use devTools with multiremote

### ISSUE THREAD

**Issue by vojtech-cerveny** [webdriverio/webdriverio#5505]
**Environment (please complete the following information):**
 - **WebdriverIO version:** 6.1.17
 - **Mode:** WDIO Testrunner
 - **If WDIO Testrunner, running sync/async:** sync/async
 - **Node.js version:** v12.13.1
 - **Browser name and version:** Chrome latest
 - **Platform name and version:** Ubuntu 18.04
 - **Additional wdio packages used (if applicable):** 
```
    "@wdio/cli": "^6.1.17",
    "@wdio/devtools-service": "^6.1.14",
    "@wdio/local-runner": "^6.1.17",
    "@wdio/mocha-framework": "^6.1.17",
    "@wdio/spec-reporter": "^6.1.14",
    "@wdio/sync": "^6.1.14",
```
**Config of WebdriverIO**
```js
  runner: 'local',
  services: ['devtools'],  
  capabilities: {
    creator: {
      capabilities: {
        browserName: 'chrome',
      },
    },
    spectator: {
      capabilities: {
        browserName: 'chrome',
      },
    },
  },
```

**Describe the bug**
It is not possible to use devTools using multiremote. If I add `cdp` fuction, it returns `creator.cdp is not a function` 

```
ERROR @wdio/devtools-service: Couldn't connect to chrome: TypeError: Cannot read property 'goog:chromeOptions' of undefined
    at findCDPInterface (/home/vojtech-cerveny/Documents/dev/bee/bee-plugin-frontend/node_modules/@wdio/devtools-service/build/utils.js:33:52)
    at DevToolsService.before (/home/vojtech-cerveny/Documents/dev/bee/bee-plugin-frontend/node_modules/@wdio/devtools-service/build/index.js:64:61)
   ....
```

**To Reproduce**
```js
/* global creator, spectator */

describe('Network monitoring', () => {
  it('should have the right title', () => {

    creator.cdp('Network', 'enable')
    creator.on('Network.responseReceived', params => {
      console.log(`Loaded ${params.response.url}`)
    })
    creator.url('http://url/')

    // I need to wait for something from Network, and parse it and use it in second remote. 
    // spectator.url('https://webdriver.io/docs/multiremote.html')
    // expect(creator).toHaveTitle('WebdriverIO · Next-gen browser and mobile automation test framework for Node.js')

    // spectator.url('https://webdriver.io/docs/multiremote.html')
    // expect(spectator).toHaveTitle('Multiremote · WebdriverIO')
  })
})
```

**Expected behavior**
It should be possible to use devtools with multiremote.

**Log**
https://gist.github.com/vojtech-cerveny/c57391969a2b8682fcfe2c164d44728d

**Additional context**

**Comment by christian-bromann** [comment]
Hey,
thanks for filing the issue. This seems indeed like bug. We would appreciate any help on this so please help us out! Have a look at our [contributing guidelines](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) and reach out if you have any questions. Cheers!

**Comment by christian-bromann** [comment]
Looked into this and found that it's not that straightforward to enable this for Multiremote. The Puppeteer implementation needs to be applied to all multiremote instances which requires a bit rewrite of the code.

**Comment by tiagocmendes** [comment]
Do you know what is the status of this issue?

**Comment by christian-bromann** [comment]
@tiagocmendes contributions are welcome!

**Comment by millcek** [comment]
@christian-bromann I am facing a similar issue with MultiRemote browser config and getting the following error.

> The @wdio/devtools-service currently only supports Chrome version 63 and up, Firefox 86 and up, and Chromium as the browserName!

Regarding what you wrote above I suppose editing [this function](https://github.com/webdriverio/webdriverio/blob/2c17d96882dd1f17017f43e23c3856b51ec4d313/packages/wdio-devtools-service/src/utils.ts#L75) is not enough, am I correct?

**Comment by christian-bromann** [comment]
> am I correct?

Not sure, I haven't looked too deep into this. Contributions are welcome!

**Comment by slavamarkovski** [comment]
@christian-bromann seems now it is possible to use devtools on remote. At lease I was able to run wdio+devtools+selenoid but in single thread only. If config has maxInstances > 1 browsers are opening locally even if remote  hub is specified in the config. Is it expected?

**Comment by christian-bromann** [comment]
Fixed in #11059 , if there is anything missing, please let me know!
